addons/hr_attendance/models/res_company.py

```diff
@@ -10,6 +10,9 @@
 class ResCompany(models.Model):
     _inherit = 'res.company'
 
+    def _default_employee_token(self):
```

Comment:
Why employee? :smile: I am not against copying the code and using it for inspiration, but then we need to adjust it, no? 

```suggestion
    def _default_company_token(self):
```